dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -623,9 +631,9 @@ def diff(self, other_profile, options=None):
 
         return diff
 
-    def report(self, remove_disabled_flag=False):
+    def report(self, remove_disabled_flag: bool = False) -> Dict:
         """Report profile attribute of class; potentially pop val from self.profile."""
-        calcs_dict_keys = self._TextProfiler__calculations.keys()
```

Comment:
I'm not sure if this was a typo because I couldn't find `self._TextProfiler__calculations` anywhere else.

Reply:
Not a typo, if you go into a base class you can still call the previous classes specific variable that way in python when it has the dunder.